x-pack/plugins/fleet/server/saved_objects/index.ts

```diff
@@ -298,6 +299,7 @@ const getSavedObjectTypes = (
         version: { type: 'keyword' },
         internal: { type: 'boolean' },
         removable: { type: 'boolean' },
+        keep_policies_up_to_date: { type: 'boolean' },
```

Comment:
Do we want to index that field?

Reply:
I am not sure. What is the alternative for a field like this?

Reply:
you can add the `index: false` property to make the field not searchable